lib/monit_app_configuration.py

```diff
@@ -28,6 +28,7 @@ def create_config_file(watch, start_cmd, stop_cmd, ports, env_vars={},
     max_memory: An int that names the maximum amount of memory that this process
       is allowed to use (in megabytes) before monit should restart it.
     syslog_server: The IP of the remote syslog server to use.
+    host: The IP of the server where this file will reside.
```

Comment:
I think the comment is not clear. What about something like: The private IP of the host to check the port. Used to reliably detect a running dev_appserver.